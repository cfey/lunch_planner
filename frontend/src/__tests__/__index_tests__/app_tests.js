import React from 'react'
import renderer from 'react-test-renderer'
import App from '../../pages/index/components/app'

it('App renders correctly', () => {
  const tree = renderer
    .create(<App />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
