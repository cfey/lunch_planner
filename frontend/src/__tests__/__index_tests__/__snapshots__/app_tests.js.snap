// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  className="MuiPaper-root-1 MuiPaper-elevation2-5 MuiPaper-rounded-2"
>
  <div
    className="MuiCardContent-root-28"
  >
    <h2
      className="MuiTypography-root-29 MuiTypography-title-35"
    >
      Registered User Counter
    </h2>
    <p
      className="MuiTypography-root-29 MuiTypography-body1-38"
    >
      Registered User: 
      <b>
        -
      </b>
    </p>
  </div>
  <div
    className="MuiCardActions-root-53"
  >
    <button
      className="MuiButtonBase-root-70 MuiButton-root-55 MuiCardActions-action-54"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="button"
    >
      <span
        className="MuiButton-label-56"
      >
        Refresh
      </span>
      <span
        className="MuiTouchRipple-root-73"
      />
    </button>
  </div>
</div>
`;
